File: week 9/task3.cpp
#include<iostream>
using namespace std;
string name;
string input;
main()
 
{
   

    cout<<"Enter a string : ";
    cin>>input;

    int length=0;

int i=0;
    while(input[i] != '\0')
    {
        
        i++;
    }
    cout<<i;
    if(i%2==0)
    {
        cout<<"true";
    }
    else
    {
      cout<<"false";
    }
}

File: lab manual 14/Task3.py
def Area(country_name,country_area):
    percent=(country_area/148940000)*100
    return percent
country_name=input("Enter the country name: ")
country_area=int(input("Enetr the country area: "))
result=Area(country_name,country_area)
print(f"{country_name} is {result:.2f}% of the world's landmass")

